### ISSUE TITLE
Where to set UPTIME_KUMA_DISABLE_FRAME_SAMEORIGIN on local instalation (no docker)

### ISSUE THREAD

**Issue by Chraftingblock** [louislam/uptime-kuma#2682]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

I want to integrate parts of the Statuspage on my Nextcloud and for that, in want to allow IFrame integration. Where do i have to change the File, to get it working

### 🐻 Uptime-Kuma Version

1.19.6

### 💻 Operating System and Arch

Ubuntu 22.04.1 LTS

### 🌐 Browser

Mozilla Firefox 109.0

### 🟩 NodeJS Version

v14.21.2

**Comment by julian-piehl** [comment]
I would suggest you add the parameter `--disable-frame-sameorigin` to the startup command of the server.

```
pm2 start server/server.js --name uptime-kuma --disable-frame-sameorigin
```

https://github.com/louislam/uptime-kuma/blob/master/server/server.js#L108

**Comment by louislam** [comment]
Maybe we should support `.env` file

**Comment by julian-piehl** [comment]
> Maybe we should support `.env` file

Good idea, this would make it simpler for non-docker and docker setups.
Maybe using [dotenv](https://www.npmjs.com/package/dotenv). By using it, you don't have to change this much. Only one line to initialize dotenv and remove args-parser.

**Comment by burtonmsp** [comment]
Hi, having the same issue on a new non-Docker install. Tried the command above and got an error:

pm2 start server/server.js --name uptime-kuma --disable-frame-sameorigin

  error: unknown option `--disable-frame-sameorigin'

**Comment by louislam** [comment]
You can use `.env` file now in `1.20.1`. It maybe easier to manage these settings.

I guess you have to add a double dash `--` in order to pass it to uptime-kuma rather than used by pm2

`pm2 start server/server.js --name uptime-kuma -- --disable-frame-sameorigin`
